src/components/organisms/Dashboard.tsx

```diff
@@ -0,0 +1,148 @@
+import React, { useState, useEffect } from 'react';
+import ReactGridLayout from 'react-grid-layout';
+import { ParentSize } from '@visx/responsive';
+
+import { Pane, MinimizeIcon, MaximizeIcon, IconButton } from 'evergreen-ui';
+
+import DashboardItem from '../molecules/DashboardItem';
+import { DashboardItemSize, DashboardItemInfo, DashboardProps } from '../../dataTypes/Dashboard/DashboardTypes';
+
+import 'react-grid-layout/css/styles.css';
+import 'react-resizable/css/styles.css';
+
+type Layout = {
+    i: string;
+    x: number;
+    y: number;
+    w: number;
+    h: number;
+};
+
+const Dashboard: React.FC<DashboardProps> = ({ items }) => {
+    const [layout, setLayout] = useState<Layout[]>();
```

Comment:
```suggestion
    const [layout, setLayout] = useState<Layout[]>([]);
```
Default to empty layout instead of `undefined`/`null`.